ui/dashboards/src/components/Dashboard/Dashboard.stories.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Just starting with the empty states in this storybook. I'm going to add more in a separate PR I'm working on.

Reply:
I wonder how many examples of the empty state we want to do

I think we have similar examples in ViewDashboard and EmptyStateDashboard, and I'm guessing they look mostly the same

What's your take / your philosophy on this kind of thing?

Reply:
We're in a weird case with our storybook where we're using it both to document AND to make it possible to test cases we can't hit in the Perses app. These are a bit of overdoing it for the former case, but really helpful to make sure things work ok in the latter case.

Reply:
On the documentation-only side, it's definitely a balancing act. Our case is especially weird because we have SO MANY different dashboard components that are kinda the same (e.g. `ViewDashboard`, `DashboardApp`, `Dashboard`).